### PR TITLE
Add deployed V3 core ETH mainnet contracts

### PR DESCRIPTION
**Pull request by ryley-o**
Update configs with deployed V3 core contract suite, as logged in: https://github.com/ArtBlocks/artblocks-contracts/pull/337

This branch has been deployed to subgraph studio as `v0.0.20`, to begin the syncing process.

### CODE REVIEW

**Review by jakerockland — APPROVED** [review]
🙏 🙏 🙏

File: config/mainnet-with-secondary.json [hunk 1]
```diff
@@ -66,14 +72,21 @@
     {
       "address": "0x4aafCE293b9B0faD169c78049A81e400f518E199",
       "startBlock": 14567785,
-      "name": "Art Blocks Flagship"
+      "name": "Art Blocks Flagship Minterf Fiter for V1 core"
```
Comment (lyaunzbe) [inline config/mainnet-with-secondary.json:75]: smol typo

File: config/mainnet.json [hunk 2]
```diff
@@ -71,14 +77,21 @@
     {
       "address": "0x4aafCE293b9B0faD169c78049A81e400f518E199",
       "startBlock": 14567785,
-      "name": "Art Blocks Flagship"
+      "name": "Art Blocks Flagship Minterf Fiter for V1 core"
```
Comment (lyaunzbe) [inline config/mainnet.json:80]: smol typo

**Review by lyaunzbe — APPROVED** [review]
lgtm, just noted two smol typos

**Pull request merged**